crates/fj-export/src/lib.rs

```diff
@@ -48,4 +67,57 @@ pub fn export(mesh: &Mesh<Point<3>>, path: &Path) -> Result<(), Error> {
     Ok(())
 }
 
-pub use threemf::Error;
+fn export_stl(mesh: &Mesh<Point<3>>, path: &Path) -> Result<()> {
+    let points = mesh
+        .triangles()
+        .map(|triangle| triangle.points)
+        .collect::<Vec<_>>();
+
+    let vertices = points.iter().map(|points| {
+        points.map(|point| {
+            [point.x.into_f32(), point.y.into_f32(), point.z.into_f32()]
+        })
+    });
+
+    let normals = points
+        .iter()
+        .map(|&points| points.into())
+        .map(|triangle: Triangle<3>| triangle.to_parry().normal())
+        .collect::<Option<Vec<_>>>()
+        .ok_or_else(|| anyhow!("Unable to compute normal"))?;
```

Comment:
Since `fj_math::Triangle` is validated on construction, we could add an infallible `normal` method to it, which would make this kind of code simpler. Or, as a more incremental step, thanks to that validation the `anyhow!` here can probably be replaced with an `unreachable!`.

Neither of those need to happen in this pull request though.